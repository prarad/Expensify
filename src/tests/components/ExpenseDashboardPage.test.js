import React from 'react';
import { shallow } from 'enzyme';
import DashboardPage from '../../components/ExpenseDashboardPage';

test('should render ExpenseDashboardPage correclty', () => {
  const wrapper = shallow(<DashboardPage />)
  expect(wrapper).toMatchSnapshot()
})