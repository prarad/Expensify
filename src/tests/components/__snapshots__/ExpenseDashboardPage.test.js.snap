// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseDashboardPage correclty 1`] = `
<div
  className="ExpenseDashboardPage"
>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
